drivers: pcie: Remove magic number and cleanup

finikorg opened the pull request:
- Use definitions instead of magic numbers.
 - Cleanup code to move reading register before actual usage

finikorg committed:
drivers: pcie: Remove magic number

Use definitions instead of magic numbers.

Signed-off-by: Andrei Emeltchenko <andrei.emeltchenko@intel.com>

jhedberg commented:
Looks fine. While you're at it, maybe add a second commit to move the reg read just before this write operation (currently the function does the read immediately, but that's a bit wasteful since there are two return branches before we need to access the value).

jhedberg commented:
Correction, *four* return branches before the value is needed

finikorg committed:
drivers: pcie: Read register before actual usage

Move reading PCIE_CONF_CMDSTAT before actual usage. There are four
return branches before value is used.

Signed-off-by: Andrei Emeltchenko <andrei.emeltchenko@intel.com>
